staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher.go

```diff
@@ -377,6 +377,8 @@ func NewCacherFromConfig(config Config) (*Cacher, error) {
 	// Configure reflector's pager to for an appropriate pagination chunk size for fetching data from
 	// storage. The pager falls back to full list if paginated list calls fail due to an "Expired" error.
 	reflector.WatchListPageSize = storageWatchListPageSize
+	// TODO: Make it configurable.
```

Comment:
If you think it's necessary, can you open an issue for it and link here?
I'm not convinced and I would just remove this TODO.

But instead add a comment explaining why we are doing it (roughly describing the motivation from the original bug).